src/pages/HomePage.js

```diff
@@ -1,13 +1,66 @@
 import React from "react";
-import { useSelector } from "react-redux";
+import "../style/App.scss";
+import "../style/HomePage.scss";
+import ExampleComponent from "react-rounded-image";
+import defaultProfileImg from "../assets/profilePicture.png";
+import { FormattedMessage } from "react-intl";
+import { Map, TileLayer } from "react-leaflet";
+import { mapConstants } from "../constants/map.constants";
+import menuIcon from "../assets/icons/menu_icon.png";
+import smilies from "../assets/smilies.png";
 
 function HomePage() {
     const { user: { email }} = JSON.parse(localStorage.getItem("user"));
-    const { message } = useSelector(state => state.alert);
+
     return (
-        <div>      
-            { message } : { email }
+        <div className="main-div">
+  
+            <div className="left-menu">
+
+                <div className="left-menu__options">
+                    <img src={menuIcon} alt="menu_button"></img>
+                </div>
+
+                <p className="left-menu__app-title">TARGET</p>
+
+                <div className="left-menu__profile-picture">
+                    <ExampleComponent
+                        image={defaultProfileImg}
+                        roundedSize="0"
+                        imageWidth="150"
+                        imageHeight="150"
+                    />
+                </div>
+
+                <p className="left-menu__username">{ email }</p>
+
+                <div className="left-menu__edit-logout">
+                    <a href="/"><FormattedMessage id="homepage.edit.text"/></a> &nbsp; / &nbsp;
+                    <a href="/"><FormattedMessage id="homepage.logout.text"/></a>
+                </div>
+
+                <hr className="left-menu__hr"></hr>
+
+                <p className="left-menu__matches"><FormattedMessage id="homepage.nomatches.text"/></p>
+
+                <div className="left-menu__smilies">
+                    <img src={smilies} alt="smilies" className="smilies-img"></img>
+                </div>
+     
         </div>
+  
+        <Map
```

Comment:
I think the map should be in a separate component

Reply:
Why is that needed?

Reply:
Is not needed, I think is better because you have each component with their own logic. And maybe you can re-use that map component in other screen